master/buildbot/www/hooks/github.py

```diff
@@ -245,11 +325,21 @@ def __init__(self, master, options):
         BaseHookHandler.__init__(self, master, options)
 
         klass = options.get('class', GitHubEventHandler)
+        klass_kwargs = {
+            'master': master,
+            'codebase': options.get('codebase', None),
+            'github_property_whitelist': options.get('github_property_whitelist', None),
+            'github_api_endpoint': options.get('github_api_endpoint', None) or 'https://api.github.com',
+            'debug': options.get('debug', None) or False,
+            'verify': options.get('verify', None) or False,
+            'oauth_token': options.get('oauth_token', None),
+            'integration_id': options.get('integration_id', None),
```

Comment:
so instead of oauth_token, user must provide integration_id, private_key and installation_id? I am still not convinced, this is easier.